knot-thing/src/main/java/com/cesar/knot_sdk/knot_state_machine/states/base_classes/State.kt

```diff
@@ -0,0 +1,34 @@
+package com.cesar.knot_sdk.knot_state_machine.states.base_classes
+
+abstract class State {
+
+    val NO_ERROR_MESSAGE : String = "no error"
+    val NETWORK_ERROR_MESSAGE : String = "no error"
+    val TIMEOUT_MESSAGE  : String = "TIMEOUT"
+    val SUCCESS_MESSAGE : String = "Success"
```

Comment:
Maybe you can change the keyword success for "Proceed" or something more appropriate.